google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/TimePartitioning.java

```diff
@@ -72,17 +158,37 @@ public Long getExpirationMs() {
     return expirationMs;
   }
 
+
+  /**
+   * If not set, the table is partitioned by pseudo column '_PARTITIONTIME'; if set, the table is
+   * partitioned by this field.
+   */
+  public String getField() {
```

Comment:
I think `getField` and `getRequirePartitionFilter` should be `@BetaApi` as well.